File: build-system/integration-test/test-projects/jetifier/app/src/main/java/com/example/app/MainActivity.java
package com.example.app;

import android.os.Bundle;
import android.support.v7.app.AppCompatActivity;
import android.widget.TextView;
import com.example.androidlib.Greetings;

public class MainActivity extends AppCompatActivity {

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_main);

        TextView textView = (TextView) findViewById(R.id.helloText);
        String text = new Greetings().getGreetings(getApplicationContext());
        textView.setText(text);
    }
}

File: build-system/integration-test/test-projects/kotlinApp/src/main/kotlin/com/example/android/kotlin/MainActivity.kt
package com.example.android.kotlin

import android.app.Activity
import android.os.Bundle
import android.widget.Button
import android.widget.TextView

class MainActivity : Activity() {

    var text: TextView? = null

    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_layout)
        text = findViewById(R.id.someText) as TextView
        if (text != null) {
            text?.setText("testing kotlin")
            android.util.Log.d("kotlin", text?.getText().toString())
        }
        val click = findViewById(R.id.click) as Button
        click?.setOnClickListener { text?.setText("clicked!") }
    }
}
